lms/templates/static_templates/contact.html

```diff
@@ -9,8 +9,15 @@
         <h1>
             <%block name="pageheader">${page_header or _("Contact")}</%block>
         </h1>
+        <p>Телесенс Академия</p>
+        <p>ул. Молочная, 38</p>
+        <p>Украина, г. Харьков, 61001</p>
+        <p><a href="tel:380577190506">+38 (057) 719 05 06</a></p>
+        <p><a href="tel:380503664009">+38 (050) 366 40 09</a></p>
+        <p><a href="tel:380632836464">+38 (063) 283 64 64</a></p>
+        <p><a href="tel:380673595495">+38 (067) 359 54 95</a></p>
         <p>
-            <%block name="pagecontent">${page_content or _("This page left intentionally blank. Feel free to add your own content.")}</%block>
+            <a href="">telesensacademy@gmail.com</a>
```

Comment:
Any changes made to the platform will affect all microsites. Do I understand correctly that all microsites will have the `/contact` page? And the same about the `.wrapper-profile` style changes.